src/controllers/__tests__/profile.test.js

```diff
@@ -52,6 +52,21 @@ describe('User routes', () => {
       expect(res.headers['content-type']).toMatch('application/json');
       expect(res.body).toEqual(expect.objectContaining(expectedResponse));
     });
+    test('If user email exists, should return an error with status code 400', async () => {
```

Comment:
The test case description and the actual code are not matching. 
Your test is supposed to return 400 error for existing email, but your implementing oldpassword update!